doc/historylog.rst

```diff
@@ -2,26 +2,48 @@
 History
 #######
 
-With the **History** pane, you can view an automatically de-duplicated, time-stamped list of every command you enter into any connected :doc:`ipythonconsole`.
+With the **History** pane, you can view all the commands entered into any :doc:`ipythonconsole` along with their time-stamp.
 
 .. image:: images/history/history-menu.png
    :alt: Spyder History Log, displaying a list of previously executed commands
 
+(Replace screenshot for screenshot without the options menu)
```

Comment:
```suggestion
```

* ~Grammar: "for" -> "with"~
* Unless its a placeholder for an image that isn't there at all, I would suggest that you use the OP, Github comments or elsewhere for meta remarks like this, so they don't pollute the repo itself and cause unnecessary problems when rebasing. Thanks.